CLI/commands/ContractClass.js

```diff
@@ -0,0 +1,32 @@
+const common = require('./common/common_functions')
+const abis = require('./helpers/contract_abis')
+const gbl = require('./common/global')
+
+class Contract {
+  static async connect (contractName, address) {
+    return new web3.eth.Contract(abis[contractName](), address)
+  }
+
+  static async call (contract, methodName, methodArguments) {
+    try {
+      return await contract.methods[methodName].apply(this, methodArguments).call()
+    } catch (e) {
+      console.log(`${methodName} call - `.toUpperCase(), e.message)
+      process.exit(0)
+    }
+  }
+
+  static async owner (contract) {
+    return await contract.methods.owner().call() == Issuer.address
```

Comment:
owner should return the contract owner. iAmOwner() should make this comparison instead.